nearcore/src/config.rs

```diff
@@ -964,19 +963,7 @@ pub fn init_configs(
                 CryptoHash::default(),
             );
             add_protocol_account(&mut records);
-            let shards = if num_shards > 1 {
-                ShardLayout::v1(
-                    (1..num_shards)
-                        .map(|f| {
-                            AccountId::from_str(format!("shard{f}.test.near").as_str()).unwrap()
-                        })
-                        .collect(),
-                    None,
-                    1,
-                )
-            } else {
-                ShardLayout::v0_single_shard()
-            };
+            let shards = ShardLayout::multi_shard(num_shards, 3);
```

Comment:
To fix the `runtime-params-estimator` test you can to set the version here to 0. It's suboptimal and definitely buggy but I don't think it's worth properly debugging this rather old test framework.

Reply:
Oh my it breaks a bunch of other tests. I guess it will be easier to fix it here after all.

Reply:
> To fix the `runtime-params-estimator` test you can to set the version here to 0.

done